release-doc-in-process/all_vf.TTL

```diff
@@ -427,6 +441,47 @@ vf:resourceConformsTo
         rdfs:range      vf:ResourceSpecification ;
         rdfs:comment    "The primary resource knowledge specification or definition of an existing or potential resource." .
 
+vf:intents
+        a                   owl:ObjectProperty ;
+        rdfs:label         "intents" ;
+        rdfs:domain         vf:Proposal ;
+        rdfs:range          vf:Intent ;
+        rdfs:comment        "The primary offers, requests, estimates, or other intents proposed." .
+
+vf:reciprocalIntents
+        a                   owl:ObjectProperty ;
+        rdfs:label         "reciprocal intents" ;
+        rdfs:domain         vf:Proposal ;
+        rdfs:range          vf:Intent ;
+        rdfs:comment        "The reciprocal offers, requests, estimates, or other intents proposed." .
+
+vf:intentOf
```

Comment:
inverse properties just complicate queries, one has to always query for both directions to make sure all results get returned. if we let any inverse property slip into the vocab we should remove it as part of next release cleanup.
on can always adjust position of object and subject in the statement without need to have inverse property, in JSON-LD `@reverse` keyword provides some convenience.